AEPConcierge/Sources/Views/ChatView.swift

```diff
@@ -244,6 +256,66 @@ public struct ChatView: View {
             viewModel.toggleMic(currentSelectionLocation: selectedTextRange.location)
         }
     }
+    
+    private func sendFeedbackFor(messageId: UUID?, with feedbackPayload: FeedbackPayload) {
+        // Update the message with the feedback sentiment
+        guard let messageId = feedbackMessageId, let index = viewModel.messages.firstIndex(where: { $0.id == messageId }) else {
+            Log.debug(label: LOG_TAG, "Unable to send feedback, the message was not retrievable from the chat.")
+            return
+        }
+        
+        // get the message information for which feedback was provided
+        var currentMessage = viewModel.messages[index]
+        
+        // attach sentiment
+        currentMessage.feedbackSentiment = feedbackPayload.sentiment
+        
+        // Write the updated message back to the array so UI updates
+        viewModel.messages[index] = currentMessage
+        
+        // get the payload
+        let payload = currentMessage.payload
```

Comment:
It looks like the payload isnt used in the event data anywhere? Sorry I'm not familiar with the intended structure of the feedback XDM so not sure if/where it should go in the hierarchy

Reply:
i think it's something i was originally using but isn't in use anymore and i forgot to remove it - removing it.